### ISSUE TITLE
configuration document?

### ISSUE THREAD

**Issue by ghpass** [abh/geodns#87]
The readme file is too simple. Where is the configuration document? Thanks!

**Comment by ianvm** [comment]
Did you check out the example zone files at 
https://github.com/abh/geodns/tree/master/dns ?

**Comment by ghpass** [comment]
yes. I had read all the examples. But I don't know how to write my own zone. By the way, I'm famliar with bind's configuration.

**Comment by amiri27** [comment]
Hi,

Same is here.. We spend many times to activate the Geolocation based on IP or ASN on Country but that didn't work at all..

Could you please give me an example of configuration? Example files didn't work for location based..

Regards,

**Comment by sharuijinfriend** [comment]
> Hi,
> 
> Same is here.. We spend many times to activate the Geolocation based on IP or ASN on Country but that didn't work at all..
> 
> Could you please give me an example of configuration? Example files didn't work for location based..
> 
> Regards,

Have you ever succeed in running geodns based on  IP or ASN on Country? I met the same problem and want your help.
